src/components/modals/LibraryCardFormModal.tsx

```diff
@@ -18,15 +19,14 @@ interface UserOption {
 }
 
 interface FormFieldRowProps {
-  card: LibraryCard;
+  selectedUsers: UserOption[];
   dropdownOptions: UserOption[];
-  updateUser: (value: SingleValue<UserOption>) => void;
-  setCard: React.Dispatch<React.SetStateAction<LibraryCard>>;
+  updateUsers: (values: MultiValue<UserOption>) => void;
 }
 
 export interface LibraryCardFormModalProps {
   showModal: boolean;
-  createOnClick: (card: LibraryCard) => void;
+  createOnClick: (cards: LibraryCard[]) => void;
   closeModal: () => void;
   users: UserOption[];
   card: LibraryCard;
```

Comment:
I realize it's not part of your change set, but the card property looks like it's safe to remove.  The test doesn't really use it either - seems to build an empty object.